src/atom-config/index.js

```diff
@@ -0,0 +1,48 @@
+'use babel'
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/extensions
+import { CompositeDisposable } from 'atom'
+import configs from './config-mappings'
+import { setValue } from './make-handler'
+import requestMigration from './migrate'
+
+// set idleCallbacks to migrate any old config settings
+requestMigration()
```

Comment:
🚨 This needs to be moved into a function, for example called from within `configSubscribe`. Right now this is running on `import` of this file!

Reply:
I don't see what's wrong with that. It sets up an idleCallback and moves on. In the case there _are_ settings that need migrating, we'd better not wait _too_ long to make them available to Atom to find time for.